src/services/refactors/addOrRemoveBracesToArrowFunction.ts

```diff
@@ -89,6 +90,14 @@ function getRefactorActionsToRemoveFunctionBraces(context: RefactorContext): rea
     return emptyArray;
 }
 
+/** @internal */
+export function addBracesToArrowFunction(file: SourceFile, returnExpression: Expression, extraStatements: readonly Statement[] = []) {

```

Comment:
I think that exporting this broke `APILibCheck` test, cause now I get this baseline change:
<details>
<summary>baseline change: `tests/baselines/reference/APILibCheck.errors.txt`</summary>

```
tsserverlibrary.internal.d.ts(24934,143): error TS2307: Cannot find module '../_namespaces/ts' or its corresponding type declarations.
tsserverlibrary.internal.d.ts(25046,139): error TS2307: Cannot find module '../_namespaces/ts' or its corresponding type declarations.
typescript.internal.d.ts(19586,143): error TS2307: Cannot find module '../_namespaces/ts' or its corresponding type declarations.
typescript.internal.d.ts(19698,139): error TS2307: Cannot find module '../_namespaces/ts' or its corresponding type declarations.


==== index.ts (0 errors) ====
    import ts = require("typescript");
    import tsInternal = require("typescript-internal");
    import tsserverlibrary = require("tsserverlibrary");
    import tsserverlibraryInternal = require("tsserverlibrary-internal");
    
==== node_modules/typescript/package.json (0 errors) ====
    {
        "name": "typescript",
        "types": "/.ts/typescript.d.ts"
    }
    
==== node_modules/typescript-internal/package.json (0 errors) ====
    {
        "name": "typescript-internal",
        "types": "/.ts/typescript.internal.d.ts"
    }
    
==== node_modules/tsserverlibrary/package.json (0 errors) ====
    {
        "name": "tsserverlibrary",
        "types": "/.ts/tsserverlibrary.d.ts"
    }
    
==== node_modules/tsserverlibrary-internal/package.json (0 errors) ====
    {
        "name": "tsserverlibrary-internal",
        "types": "/.ts/tsserverlibrary.internal.d.ts"
    }
    
``` 
</details>

I'm not sure how to fix this though. My gut tells me that @jakebailey might know :P

Reply:
You need to explicitly type the return type. Check the line of the output where the error is and you'll see the bad node from our very naive dts bundler.

Reply:
Thanks for the tip! I’ll apply the suggestion soon. I wonder though - shouldnt this just get stripped out entirely anyway since its marked as internal?

Reply:
The bundler produces both public and internal outputs, since there are packages like byots and ts-expose-internals which need that. Before modules, they'd just build and disable stripInternal, but post modules, you have to use the bundler. But since the bundler is very naive, if we didn't verify that it worked with the internal exports, those users would end up with randomly broken files that could only be fixed upstream here.